CHANGELOG.md

```diff
@@ -6,6 +6,8 @@
 
 - Remove all code related to Sentry's previous Metrics product. Read more about the end of the Metrics beta ([here](https://sentry.zendesk.com/hc/en-us/articles/26369339769883-Metrics-Beta-Ended-on-October-7th)) ([#914](https://github.com/getsentry/sentry-go/pull/914))
 
+- Every integration is now a separate module, reducing the binary size and number of dependencies. Once you update `sentry-go` to latest version, you'll need to `go get` the integration you want to use. For example, if you want to use the `echo` integration, you'll need to run `go get github.com/getsentry/sentry-go/echo` ([#919](github.com/getsentry/sentry-go/pull/919)).
```

Comment:
> reducing the binary size

This should not be the case due to [Module Graph Pruning](https://go.dev/ref/mod#graph-pruning).

Reply:
Hmm, I tested this locally just to make sure. Imported `echo` only, and did `go build .`

echo-new                                4931.21 KB
echo-old                                  4948.81 KB

Reply:
`go build -ldflags " -s -w" .`

```
echo-new                                  3454.99 KB
echo-old                                  3472.00 KB
```